antlr/SchemaParser.ts

```
@@ -643,513 +773,1478 @@ export class SchemaParser extends Parser {
+		} else {
+			return visitor.visitChildren(this);
+		}
+	}
+}
+
+
+export class TypeNameContext extends ParserRuleContext {
+	public IDENTIFIER(): TerminalNode { return this.getToken(SchemaParser.IDENTIFIER, 0); }
+	constructor(parent: ParserRuleContext | undefined, invokingState: number) {
+		super(parent, invokingState);
+	}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)